lib/search.js

```diff
@@ -222,6 +226,7 @@ module.exports = function(appConfig, date_field) {
                 return;
             }
         });
```

Comment:
I think you need a catch here to handle errors coming back from the API and the error handling in the then block should move.